sessions/smoother-collaboration.qmd

```diff
@@ -467,8 +538,8 @@ styler::style_dir()
 ```
 
 The thing to note, though, is that `{styler}` isn't perfect, so you
```

Comment:
```suggestion
```

I see what you were asking about reformat vs style. Yea, this section is a bit confusing, so we just remove it.